tamr_toolbox/filesystem/cloud.py

```diff
@@ -0,0 +1,76 @@
+from google.cloud import storage
+from mypy_boto3_s3.client import S3Client
+
+
+def gcs_upload(
+    cloud_client: storage.Client,
```

Comment:
With the previous suggested change, type hints involving the optional modules need to be specified as strings like so:
```suggestion
    cloud_client: "storage.Client",
```

Reply:
This will also need to be done in the other function definitions in this file, including the S3 ones.